commons/src/main/java/org/sonarsource/sonarlint/core/commons/IssueStatus.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * SonarLint Core - Commons
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarsource.sonarlint.core.commons;
+
+public enum IssueStatus {
+  CONFIRM("Confirm", "confirm"),
+  UN_CONFIRM("Unconfirm", "unconfirm"),
+  REOPEN("Reopen", "reopen"),
+  RESOLVE("Resolve", "resolve"),
+  FALSE_POSITIVE("False positive", "falsepositive"),
+  WONT_FIX("Won't fix", "wontfix"),
+  CLOSE("Close", "close"),
+  SET_IN_REVIEW("In Review", "setinreview"),
+  RESOLVE_AS_REVIEWED("Reviewed", "resolveasreviewed"),
+  RESET_AS_TO_REVIEW("To Review", "resetastoreview");
```

Comment:
I don't think you need to have all those values, couldn't you keep only the 2 that are needed (WF & FP)?

Reply:
Addressed in [e3eedf4](https://github.com/SonarSource/sonarlint-core/pull/612/commits/e3eedf4a8d30bebab82f68b85844e20e2314b3f0)